src/coreComponents/physicsSolvers/fluidFlow/ThermalCompositionalMultiphaseBaseKernels.hpp

```diff
@@ -599,7 +601,7 @@ class ScalingForSystemSolutionKernel : public isothermalCompositionalMultiphaseB
     {
       // compute the change in temperature
       real64 const temp = m_temperature[ei];
-      real64 const absTempChange = LvArray::math::abs( m_localSolution[stack.localRow + m_numComp + 1] );
+      real64 const absTempChange = LvArray::math::abs( m_localSolution[stack.localRow + m_temperatureOffset] );
```

Comment:
this change looks strange to me...is this OK?

Reply:
Agree.  the change was needed so that wells could use the scaling kernel implemented for the reservoir.  The location of the temperature variable is different for well and reservoir, hence the offset variable.  It's ok but fragile.  Another PR could see this being replaced with a traits class, but defining/changing lineups with traits classes would probably need more vetting beyond this kernel.